.github/workflows/push.yml

```diff
@@ -62,7 +62,7 @@ jobs:
       - uses: actions/checkout@v3
 
       - name: Get Image Tag
-        run: echo "::set-output name=tag::sha-${GITHUB_SHA::7}"
+        run: echo "tag=sha-${GITHUB_SHA::7}" >>$GITHUB_OUTPUT
```

Comment:
I think this would be better in a separate PR.

Reply:
Moved to https://github.com/anexia-it/k8s-anexia-ccm/pull/188